### DISCUSSION TITLE
Home Assistant snapshot saved directory.

### DISCUSSION THREAD

**Discussion by tntn04** [mrlt8/docker-wyze-bridge#819] [General]
Hello I just updated from Docker Wyze Bridge 1.11.10 to 2.1.8 and noticed that my snapshot is no longer saved in /config/www/.  It now saves the snapshot in /media/wyze/img which I can't access using the camera integration in home assistant.  Is this suppose to happen?  Reading the documentation saving in img is when you run it as a separate container and when you run it as a Home Assistant addon it should be saved in /config/www.

**Comment by mrlt8** [comment]
This was changed here #660. 
You can still specify your own directory using `IMG_DIR` or access the snapshot directly using:
`http://homeassistant.local:5000/snapshot/<cam_name>.jpg`

**Reply by tntn04** [reply]
Thank you for the quick response.  This fixed it for me.  Perhaps you can update the HA wiki page to reflect this.  From the Wiki at https://github.com/mrlt8/docker-wyze-bridge/wiki/Home-Assistant.
Still Images/Snapshots
The add-on should automatically grab a thumbnail for each of your cameras from wyze and save it to /config/www/cam-name.jpg which should be accessible at http://127.0.0.1:8123/local/cam-name.jpg
